<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contacts</class>
 <widget class="QDialog" name="contacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>联系人</string>
  </property>
  <property name="windowIcon">
   <iconset resource="souce.qrc">
    <normaloff>:/souce/souce/登录.png</normaloff>:/souce/souce/登录.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="contactsBack" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>5</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;通讯录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contClose">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#contClose{
	background-color:tomato;
	border-radius:10px;}
#contClose:hover{border-image: url(:/souce/souce/关闭1.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="contMin">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">#contMin{
background-color:rgb(255,224,81);
	border-radius:10px;}
#contMin:hover{
	border-image: url(:/souce/souce/最小化.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="contactsBox" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1000</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <widget class="QWidget" name="contactUser" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243,243,243);
</string>
    </property>
    <widget class="QLabel" name="contUserLogo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/souce/souce/登录.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="contUserMail">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#contUserMail{
	text-align:center;
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;用户账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="contactINfo" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>600</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219,219,219);</string>
    </property>
    <widget class="QGroupBox" name="contInfoTop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>160</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="contLogo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/souce/souce/登录.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="contName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="contEmail">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>邮箱</string>
      </property>
     </widget>
     <widget class="QPushButton" name="contWriteLetter">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>115</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contWriteLetter{
	border:none;
	font-size:14px;
color:white;
	background-color: rgb(29, 171, 123);
}
#contWriteLetter:hover{
background-color: rgb(26, 227, 159);
}</string>
      </property>
      <property name="text">
       <string>写信</string>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/发送.png</normaloff>:/souce/souce/发送.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="delContact">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#delContact{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#delContact:hover{
background-color:tomato;
color:white;
}</string>
      </property>
      <property name="text">
       <string>删除联系人</string>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/关闭1.png</normaloff>:/souce/souce/关闭1.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="contInfoBottom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>179</y>
       <width>581</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>基本信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>联系电话:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>社交信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>QQ:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>WeChat:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contRemarkName">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>69</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contPhone">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contQQ">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contWechat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="contCancel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>312</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contCancel{
	border:none;
	font-size:14px;
color:white;
	background-color: rgb(29, 171, 123);
}
#contCancel:hover{
background-color: rgb(26, 227, 159);
}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="contSave">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>312</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#contSave{
	border:none;
	font-size:14px;
color:white;
	background-color: rgb(29, 171, 123);
}
#contSave:hover{
background-color: rgb(26, 227, 159);
}</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="contactsList">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>0</y>
      <width>241</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="gridSize">
     <size>
      <width>200</width>
      <height>65</height>
     </size>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>q</string>
     </property>
     <property name="icon">
      <iconset>
       <selectedon>:/souce/souce/登录.png</selectedon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>adsadasda</string>
     </property>
     <property name="icon">
      <iconset>
       <selectedon>:/souce/souce/登录.png</selectedon>
      </iconset>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="souce.qrc"/>
 </resources>
 <connections/>
</ui>
